test/unit/BondingManager.js

```diff
@@ -375,6 +371,26 @@ contract("BondingManager", accounts => {
                     await bondingManager.bond(2000, transcoder2, {from: delegator})
                     assert.equal((await bondingManager.getTranscoder(transcoder0)).deactivationRound, currentRound + 2)
                 })
+
+                it("fires a TranscoderActivated event for the new transcoder", async () => {
+                    const txRes = await bondingManager.bond(2000, transcoder2, {from: delegator})
+                    truffleAssert.eventEmitted(
+                        txRes,
+                        "TranscoderActivated",
+                        e => e.transcoder == transcoder2 && e.activationRound == currentRound +2,
+                        "TranscoderActivated event not emitted correctly"
+                    )
+                })
+
+                it("fires a TranscoderDeactivated event for the evicted transcoder", async () => {
```

Comment:
Since we already test for this in the `it` block starting on L350 we should be able to remove this test.

Reply:
fixed in 0e498c2